static/js/apps/explore/topics.json

```diff
@@ -10,16 +10,34 @@
   "topics": {
     "economy": {
       "title": "Economy",
-      "description": "",
+      "description": "Understanding economics illuminates the intricate web of human choices and resource allocation, guiding governments, businesses, and individuals to make informed decisions for a prosperous and sustainable future. Search for some economic data or click around on one of the suggested searches below to get started.",
       "examples": {
-        "general": [],
-        "specific": [],
-        "comparison": []
+        "general": [
+          "How many businesses are there in the US?",
+          "Most common jobs in Texas",
+          "Median income in Santa Clara County",
+          "What are the top industries in the US?",
+          "How many people in California struggle with poverty?",
+          "US global economic activity"
+        ],
+        "specific": [
+          "How much does the US spend on its military?",
+          "Number of privately owned businesses in California"
+        ],
+        "comparison": [
+          "Income vs health insurance rates in USA"
+        ]
       },
       "meta": {
-        "dataPointCount": 7891011,
-        "variableCount": 3456,
-        "sources": ["United Nations"]
+        "dataPointCount": 0,
+        "variableCount": 67486,
+        "sources": [
+          "U.S. Bureau of Economic Analysis",
+          "U.S. Bureau of Labor Statistics",
+          "U.S. Census Bureau",
+          "U.S. Department of Labor",
+          "World Bank"
```

Comment:
I think we have other sources for economy too?

Eurostat, RBI (India), etc.  Ditto for other sources.

Generally, given these are broad verticals, I feel we might be able to find a way to fit most of our sources listed here -- https://autopush.datacommons.org/browser/Source ?

@n-h-diaz for further consultation

Reply:
Do we want to list them all? The examples on the pages are very US centric, which is why I filtered to those sources.

Reply:
@rvguha wanted to list them all, double check with him?